overture-tiles-cdk/README.md

```diff
@@ -11,3 +9,20 @@ The `cdk.json` file tells the CDK Toolkit how to execute your app.
 * `npx cdk deploy`  deploy this stack to your default AWS account/region
 * `npx cdk diff`    compare deployed stack with current state
 * `npx cdk synth`   emits the synthesized CloudFormation template
+
+## Prerequisites
+- [AWS CLI](https://docs.aws.amazon.com/cli/)
+- [AWS CDK](https://aws.amazon.com/cdk/)
+  - `npm install -g aws-cdk`
+
+## Deploying
+- Update configuration in `/lib/config.ts`
```

Comment:
should this be `bin/overture-tiles-cdk.ts`? 

Is there a way to store this account-specific configuration outside of Git?

Reply:
```
should this be bin/overture-tiles-cdk.ts?
```
Thanks for noticing this. Will update.

```
Is there a way to store this account-specific configuration outside of Git?
```
One option is to leverage environment variables and/or cloudformation outputs. Upon stack creation, the CDK could require an `$OVERTURE_TILES_BUCKET_NAME` env variable to be set. The stack would then export the bucket name so that future deployments do not require `$OVERTURE_TILES_BUCKET_NAME`. Removing the need to provide the configuration every deployment would likely be less error prone, especially when multiple maintainers are managing the same stack.

Any thoughts or other ideas?

Reply:
I think the config in the .ts file is fine for now.